pom.xml

```diff
@@ -37,14 +37,32 @@
               <!-- stream benchmark under BSD-like license with a few exceptions around benchmarking results -->
               <exclude>tools/bin/src/stream/*</exclude> 
 
-              <!-- PyGreSQL an open-source Python module that interfaces to a PostgreSQL database under the Python license -->
+              <!-- exclude open-source python modules -->
+              <!-- MIT license for open-source python modules -->
               <exclude>tools/bin/pythonSrc/PyGreSQL-4.0/**</exclude>
+              <exclude>tools/bin/pythonSrc/PSI-0.3b2_gp/**</exclude>
+              <exclude>tools/bin/pythonSrc/lockfile-0.9.1/**</exclude>
+              <!-- BSD license for open-source python modules -->
+              <exclude>tools/bin/pythonSrc/pychecker-0.8.18/**</exclude>
+              <exclude>tools/bin/pythonSrc/unittest2-0.5.1/**</exclude>
```

Comment:
unittest2-0.5.1/PKG-INFO says License is unknown. Please provide support for your claim that this is a BSD licensed code.